### PR TITLE
Document the creation of fake devices

### PR DESCRIPTION
**Pull request by lbonn**
This is my method for really easy fake devices.

Would appreciate some proofreading (typos and going through the steps).

### CODE REVIEW

File: docs/running-locally.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This applies if you need to run Aktualizr from another Linux distribution, or OSX (experimental, might need more work).
+
+Follow the link:../README.adoc#building[building instructions from the README].
+
+Aktualizr can then be run from the CMake build directory: `./build/src/aktualizr_primary/aktualizr` or installed from CMake: `make install` after setting `DESTDIR`.
+
+== Prepairing an environment
+
+This setup will help you running Aktualizr in isolation. Multiple instances can be run on the same machine if they use local configuration tied up to a local directory.
```
Comment (patrickvacek) [inline docs/running-locally.adoc]: Just "tied to a local directory".

File: docs/running-locally.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+To use virtual secondaries as well, store individual JSON configuration files in a `secondary_configs` subdirectory. link:../config/secondary/virtualsec.json[A sample configuration from the git repository] can be used as-is for one secondary.
+
+In a multiple secondaries case, the different configurations will need to refer to distinct subdirectories (see `full_client_dir`, `firmware_path`...).
+
+== Usage
+
+`aktualizr -c .` is the common invocation which will pick up the configuration from the local `conf.toml`.
+Without argument it will run as a self-standing automatic service. For development purposes, it is often useful to trigger Aktualizr selectively (see link:./selectively-triggering-aktualizr.adoc[]).
+
+To stop using one device and make Aktualizr do a re-provision, remove the `storage` directory from the environment directory when it is not running and launch it again.
```
Comment (patrickvacek) [inline docs/running-locally.adoc]: "To stop using one device and force Aktualizr to re-provision..."

File: docs/selectively-triggering-aktualizr.adoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Aktualizr normally runs as a `systemd` service and regularly polls for updates. The default polling interval is set at 10s, and this should be increased by a few orders of magnitude in a production system.
 
-It is also possible to trigger the update cycle, or individual parts of the update cycle, manually or programmatically. This is done by using aktualizr's `--running-mode` option (or by setting `running_mode` in the `[uptane]` configuration section).
+It is also possible to trigger the update cycle, or individual parts of the update cycle, manually or programmatically. This is done by using aktualizr's `--running-mode` option, or just supplying the operation name as a subcommand.
```
Comment (patrickvacek) [inline docs/selectively-triggering-aktualizr.adoc]: You still can use the configuration option for now, but I'd like to remove it soon, so it's fine with me to remove this.

Comment (lbonn) [reply docs/selectively-triggering-aktualizr.adoc]: Yes, as we still don't know why would one use it, I thought of removing it from the docs first.

Comment (lbonn) [reply docs/selectively-triggering-aktualizr.adoc]: Well, I didn't really look for that in particular, I was only documenting the "subcommand" syntax. It was convenient to just ditch it as well for clarity.

**Pull request merged**